RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusterECALOnly_cfi.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEcal.EgammaClusterProducers.particleFlowSuperClusterECALMustache_cfi import particleFlowSuperClusterECALMustache as _particleFlowSuperClusterECALMustache
+
+# "Mustache" clustering
+particleFlowSuperClusterECALOnly = _particleFlowSuperClusterECALMustache.clone(
+    # ECAL-only (no primary vertices) regression setup
+    regressionConfig = _particleFlowSuperClusterECALMustache.regressionConfig.clone(
+        isHLT = cms.bool(True),
+        eRecHitThreshold = cms.double(1.),
+        regressionKeyEB  = cms.string('pfscecal_EBCorrection_online'),
+        uncertaintyKeyEB = cms.string('pfscecal_EBUncertainty_online'),
+        regressionKeyEE  = cms.string('pfscecal_EECorrection_online'),
+        uncertaintyKeyEE = cms.string('pfscecal_EEUncertainty_online'),
+        vertexCollection = cms.InputTag(''),
+    ),
+    # ECAL-only (no primary vertices) thresholds
+    thresh_PFClusterBarrel = cms.double(0.5),
+    thresh_PFClusterEndcap = cms.double(0.5),
+    thresh_PFClusterES = cms.double(0.5),
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

in the above it will be ```thresh_PFClusterES = 0.5``` and so on

Reply:
Right, sorry, I forgot to fix that.

I don't know if I can make further changes to this PR as I'm now using the same branch in the Patatrack developments.
If you consider this a strict requirement, I'll see what I can do.

Reply:
I'm not sure about all the implications wrt using this branch in Patatrack developments.
I can suggest a solution to open a replacement/followup PR based on this PR changes + the pending changes from this review on a branch different from this PR topic branch.

Reply:
Honestly, we always come to the point where I believe the expectations you have from other people are not in line with the amount of work they are willing to do.

I believe I have done more changes and cleaning up in this PR as I was supposed to do, and the rest should be handled by the ECAL DPG reco contacts and/or by the RECO conveners.

So what I am going to do is to make this last fix, and expect the PR to be signed by RECO immediately afterwards.